test/execution/test_plan.py

```diff
@@ -640,6 +640,7 @@ def forward(self, x):
     # Compare with local plan
     assert th.all(decrypted - expected.detach() < 1e-2)
     # assert fetched_plan.state.state_placeholders != plan.state.state_placeholders #TODO
+    assert str(fetched_plan.state.tensors()) == str(plan.state.tensors())
```

Comment:
Do we have to compare string representations here? Could we compare the state tensors directly?